frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/MembersPage/MembersPage.tsx

```diff
@@ -286,6 +286,13 @@ const MembersPage = ({ location, router }: IMembersPageProps): JSX.Element => {
             setCreateUserErrors({
               email: "A user with this email address has already been invited",
             });
+          } else if (
+            userErrors.data.errors?.[0].reason.includes("password too long")
+          ) {
+            renderFlash(
```

Comment:
why `renderFlash` instead of `setCreateUserErrors({password: "..."});`

Reply:
Was just going off of the figma, which shows this as that error banner instead of error around the password box. I can definitely add a break tho!

Reply:
Figma in question: https://www.figma.com/file/UqduTVi5mQtCWUjwDwMn32/%2315625-Update-maximum-password-and-improve-error-message?type=design&node-id=2-130&mode=design&t=Al3jHJWp1OlF3AWp-0

Reply:
cc @noahtalerman is a line break like this OK? 
![Screenshot 2024-01-03 at 11 08 45 AM](https://github.com/fleetdm/fleet/assets/19838370/66b9ac37-886f-43fe-8ade-3d7ee3fc2f07)

Reply:
@jahzielv shoot, the UI w/ and w/o linebreak isn't great. Good catch @jacobshandling.

I think we need to make a tweak. Sorry about changing things up on us.

I think let's go w/ the UI change in Figma [here]((https://www.figma.com/file/UqduTVi5mQtCWUjwDwMn32/%2315625-Update-maximum-password-and-improve-error-message?type=design&node-id=314%3A66&mode=design&t=QDZkyQUMkJpwmYfB-1) instead. We won't give the user a hint to the auth_salt_key_size docs which is ok.

I updated the "To fix" in the issue description to reflect this: #15625

Reply:
Done!